<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>392</width>
     <height>400</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_login">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>280</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="commit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>280</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tel">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>60</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>140</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>61</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_register">
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QLineEdit" name="le_password_register">
     <property name="geometry">
      <rect>
       <x>159</x>
       <y>150</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_3">
     <property name="geometry">
      <rect>
       <x>229</x>
       <y>270</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tel_register">
     <property name="geometry">
      <rect>
       <x>159</x>
       <y>51</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>输入11位手机号</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password_register">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="commit_3">
     <property name="geometry">
      <rect>
       <x>299</x>
       <y>270</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tel_register">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password_cofirm">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>200</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password_cofirm">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_noname">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>100</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>昵称</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_noname">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_icon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>208</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resource.qrc">:/images/rgh.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_tel_register</tabstop>
  <tabstop>le_noname</tabstop>
  <tabstop>le_password_register</tabstop>
  <tabstop>le_password_cofirm</tabstop>
  <tabstop>commit_3</tabstop>
  <tabstop>clear_3</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>le_tel</tabstop>
  <tabstop>le_password</tabstop>
  <tabstop>commit</tabstop>
  <tabstop>clear</tabstop>
 </tabstops>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
